### DISCUSSION TITLE
[BUG] Get-PnPList error List 'SitePages' does not exist at site with URL 'https://<tenant>.sharepoint.com

### DISCUSSION THREAD

**Discussion by anthonywhite** [pnp/powershell#3052] [Q&A] [answered]
### Reporting an Issue or Missing Feature
Get-PnPList using -Identity=URL option
```List 'SitePages' does not exist at site with URL 'https://<tenant>.sharepoint.com```

### Expected behaviour 
List should be returned in same way as -Identity=guid or -Identity=Title. E.g. ```Get-PnPList -Identity 'Site Pages'``` returns

```
Title      Id                                   Url
-----      --                                   ---
Site Pages 1b71b2a4-48eb-43a9-8bfe-d9fd78325c1a /sites/sitename/SitePages
```

### Actual behaviour 
Error: ```'Get-PnPList: List 'SitePages' does not exist at site with URL'```

### Steps to reproduce behaviour
```Get-PnPList -Identity SitePages```

### What is the version of the Cmdlet module you are running?
2.1.1

### Which operating system/environment are you running PnP PowerShell on?
Windows

**Comment by veronicageek** [comment]
@gautamdsheth - I can reproduce that issue on any site (not necessarily the root site) even with latest 2.1.25. Not sure if this is also related to issue #2999 despite the merged PR.

**Comment by anthonywhite** [comment]
Thanks @veronicageek, I searched for _Open_ issues with my error message so didn't find that! Note to self...

**Comment by gautamdsheth** [comment]
Works for me with the latest nightly builds, can't reproduce it.
It was there in `2.1.1` but the nightly builds fixed it, can you please try again with a clean install of latest nightly builds ?

**Comment by veronicageek** [comment, marked as answer]
Indeed very odd, it works now.
@anthonywhite - Please update the module to the latest and try again, can't repro now. Moving to a discussion in case something is happening specifically on your machine.

![image](https://user-images.githubusercontent.com/25181757/234042500-e2e88ff0-b17f-40b7-a1cb-cf25d80e2e00.png)

**Reply by anthonywhite** [reply]
@veronicageek @gautamdsheth ```2.1.26``` works ok for me, thx. Just to check then, you believe this is caused by some Microsoft server-side change to CSOM? And the only fix needed was to PowerShell, but Framework is unaffected?
